packages/viewer/src/components/viewer-measurements/readme.md

```diff
@@ -186,55 +189,55 @@ be cloned and added to the component.
 
 ## Events
 
-| Event                | Description                                                                                        | Type                                                      |
-| -------------------- | -------------------------------------------------------------------------------------------------- | --------------------------------------------------------- |
-| `measurementAdded`   | Dispatched when a new measurement is added, either through user interaction or programmatically.   | `CustomEvent<HTMLVertexViewerDistanceMeasurementElement>` |
-| `measurementRemoved` | Dispatched when a new measurement is removed, either through user interaction or programmatically. | `CustomEvent<HTMLVertexViewerDistanceMeasurementElement>` |
+| Event                | Description                                                                                        | Type               |
+| -------------------- | -------------------------------------------------------------------------------------------------- | ------------------ |
+| `measurementAdded`   | Dispatched when a new measurement is added, either through user interaction or programmatically.   | `CustomEvent<any>` |
+| `measurementRemoved` | Dispatched when a new measurement is removed, either through user interaction or programmatically. | `CustomEvent<any>` |
 
 
 ## Methods
 
-### `addMeasurement(measurement: Measurement) => Promise<HTMLVertexViewerDistanceMeasurementElement>`
+### `addMeasurement(measurement: Measurement) => Promise<HTMLVertexViewerMeasurementDistanceElement>`
 
 Adds a new measurement as a child to this component. A new measurement
 component will be created from the template specified by
 `distance-template-id` or if undefined a default element will be created.
 
 #### Returns
 
-Type: `Promise<HTMLVertexViewerDistanceMeasurementElement>`
+Type: `Promise<any>`
```

Comment:
Looks like this typing didn't generate correctly, not sure what would have caused that